plugins/terminal/frr.py

```diff
@@ -23,7 +23,9 @@
 import re
 
 from ansible.errors import AnsibleConnectionFailure
-from ansible.plugins.terminal import TerminalBase
+from ansible_collections.ansible.netcommon.plugins.plugin_utils.terminal_base import (
```

Comment:
Isn't this also needed for Cliconf?

Reply:
@NilashishC  Added it, Thanks I missed it earlier.